menus/AdvancedControls.cpp

```diff
@@ -142,6 +145,7 @@ UI_AdvControls_Init
 void CAdvancedControls::_Init( void )
 {
 	banner.SetPicture( ART_BANNER );
+	banner.SetCoord( 72, 30 );
```

Comment:
Isn't banner supposed to have fixed position at this time?

Reply:
If I had not changed the coordinates, then the line "Input Devices" would go to the border of this banner

Reply:
It's better to keep banner at same height but move other options lower, if possible.

Reply:
Ok, I found extra space for a new checkbox without moving the banner. Check out the new commit.